packages/contracts/package.json

```diff
@@ -23,7 +23,7 @@
   "scripts": {
     "forge": "forge",
     "build": "forge build",
-    "test": "forge test --fork-url https://rpc.ankr.com/eth",
+    "test": "forge test --fork-url https://rpc.ankr.com/eth -vvv",
```

Comment:
We don't want any verbosity to be added here. That would be fine to display only summary in ci/cd. If you want to see the failing traces, you can just run the following command: `yarn workspace @ubiquity/contracts forge test --fork-url -vvv`.

Reply:
Yeah, it was just for test purpose on github, it takes long time to fork test locally, so wanted to check the error on github ci. Going to remove it after testing